charts/dapr/README.md

```diff
@@ -172,22 +171,23 @@ The Helm chart has the follow configuration options that can be supplied:
-| `dapr_scheduler.securityContext.runAsNonRoot`  | Boolean value for `securityContext.runAsNonRoot`. You may have to set this to `false` when running in Minikube | `true`                                      |
-| `dapr_scheduler.securityContext.fsGroup`       | Integer value for `securityContext.fsGroup`. Useful for adding the Scheduler process to the file system group that can write to the mounted database volume.              | `65532`                                      |
-| `dapr_scheduler.resources`                     | Value of `resources` attribute. Can be used to set memory/cpu resources/limits. See the section "Resource configuration" above. Defaults to empty                         | `{}`                                         |
-| `dapr_scheduler.debug.enabled`                 | Boolean value for enabling debug mode                                                                                                                                     | `{}`                                         |
-| `dapr_scheduler.statefulsetAnnotations`        | Custom annotations for Dapr Scheduler Statefulset                                                                                                                         | `{}`                                         |
-| `dapr_scheduler.service.annotations`           | Custom annotations for "dapr-scheduler-server" Service resource                                                                                                           | `{}`                                         |
-| `dapr_scheduler.extraEnvVars`                  | Dictionary (key: value pairs) to use as extra environment variables in the injected sidecar containers (e.g. `my-env-var: "my-val"`, etc)                                 | `{}`                                         |
+| Parameter                                     | Description                                                                                                                                                  | Default                                      |
+|-----------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------|
+| `dapr_scheduler.ha`                           | If set to true, deploys the Scheduler service with 3 nodes (this can be overridden if `dapr_scheduler.replicaCount` is bigger).                              | `false`                                      |
+| `dapr_scheduler.replicaCount`                 | Number of replicas (should be an odd number of replicas for consensus as it depends on etcd https://etcd.io/docs/v3.3/faq/)                                  | `1`  (`3` on HA mode)                        |
```

Comment:
We don't want replicaCount for scheduler because of the requirements for etcd. We will use ha for now. We want to consider "stateless" schedulers - meaning that it runs jobs but does not run etcd, but it is something for 1.15. At this version, it is 1 or 3 only.